src/kata4.humanCatDogYears.js

```diff
@@ -1,3 +1,21 @@
-const humanCatDogYears = number => {};
+const humanCatDogYears = number => {
+  let catYear = number;
```

Comment:
If you wanted you don't have to assign a value to catYear and dogYear yet as you are going to overwrite it later in the function. You could just have `let catYear;` and `let dogYear;`